### ISSUE TITLE
Build error message when project name is not equal to projectID (IEP-649)

### ISSUE THREAD

**Issue by DigKleppe** [espressif/idf-eclipse-plugin#489]
In root CMakeLists.txt:
No error:
```
# The following lines of boilerplate have to be in your project's CMakeLists
# in this exact order for cmake to work correctly
cmake_minimum_required(VERSION 3.5)

# (Not part of the boilerplate)
# This example uses an extra component for common functions such as Wi-Fi and Ethernet connection.
set(EXTRA_COMPONENT_DIRS $ENV{IDF_PATH}/examples/common_components/protocol_examples_common)

include($ENV{IDF_PATH}/tools/cmake/project.cmake)

get_filename_component(ProjectId ${CMAKE_CURRENT_LIST_DIR} NAME)
string(REPLACE " " "_" ProjectId ${ProjectId})
project(${ProjectId})
```

Build error popup without reason:
`project("test")`

Dig

**Comment by sigmaaa** [comment]
hi @DigKleppe,

Thank you for reporting this issue. The build mentioned in this [answer](https://github.com/espressif/idf-eclipse-plugin/issues/485#issuecomment-1041422212) fixes this problem. We will include these changes in the next release

**Comment by kolipakakondal** [comment]
This is fixed with v2.4.2
